src/test/java/software/amazon/documentdb/jdbc/query/DocumentDbQueryMappingServiceDateTimeTest.java

```diff
@@ -798,6 +798,32 @@ void testWhereTimestampAdd() throws SQLException {
                 operations.get(3));
     }
 
+    @Test
+    @DisplayName("Tests TIMESTAMPADD on right in WHERE clause.")
+    void testWhereTimestampAddOnRight() throws SQLException {
```

Comment:
can we also add test for TIMESTAMPADD on left?

Reply:
OK